AmericanExpress/src/main/java/com/braintreepayments/americanexpress/AmericanExpressAnalytics.kt

```diff
@@ -1,4 +1,4 @@
-package com.braintreepayments.api
+package com.braintreepayments.americanexpress
```

Comment:
random: I wonder if we want to keep `com.braintreepayments.api` prefix since it's the group ID for the artifacts in Maven Central?

Reply:
Yeah, that's a good call. Let me rename to `com.braintreepayments.api.americanexpress`.

I was originally thinking that having `api` would be redundant, but I think we should stay aligned with the gradle group ID.

Reply:
➕ 1 to keeping the group ID

Reply:
i updated the package name to `com.braintreepayments.api.americanexpress`!